framework/wazuh/wdb.py

```diff
@@ -61,9 +60,9 @@ def __send(self, msg):
 
         # Get the data size (4 bytes)
         data = self.__conn.recv(4)
-        data_size = struct.unpack('<I',data[0:4])[0]
+        data_size = struct.unpack('<I', data[0:4])[0]
 
-        data = self.__conn.recv(data_size).decode('utf-8').split(" ", 1)
+        data = self.__conn.recv(data_size).decode('utf-8', 'ignore').split(" ", 1)
```

Comment:
Key word params help self-documenting the code: `decode(encoding='utf-8', errors='ignore')`